blog/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from blog.views import BlogListView, BlogDetailView, BlogCreateView, BlogUpdateView, BlogDeleteView
+
+from blog.apps import BlogConfig
+
+app_name = BlogConfig.name
+
+urlpatterns = [
+    path('home/', BlogListView.as_view(), name='home'),
```

Comment:
Быть может все таки блог?
```suggestion
    path('blog/', BlogListView.as_view(), name='blog-list'),
```